docs/mev/faqs.md

```diff
@@ -0,0 +1,26 @@
+# FAQs
+
+1. Do builders fetch the in-turn proposer's GasCeil to build block？
+
+   Yes, you could using RPC mev_params to query validator's MEV information
+   before building block, it can help to 1) calculate a valid header with gas no
+   more than GasCeil; 2) calculate the left bidding time by
+   BidSimulationLeftOver; 3) calculate suitable builderFee by
+   validatorCommission.
+
+
+2. How does the validator choose the best bid?
+
+   The block reward is calculated as **gasFee**, the validator reward is
+   calculated as **gasFee\*(1 - commissionRate) - tipToBuilder.** Every
```

Comment:
not (1-commissionRate)

Reply:
**gasFee\*commissionRate - builderFee.**

Reply:
fixed